zcash_client_sqlite/src/wallet/init/migrations/shardtree_support.rs

```diff
@@ -173,7 +173,7 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                                 nonempty_frontier.clone(),
                                 Retention::Checkpoint {
                                     id: block_height,
-                                    is_marked: false,
+                                    marking: Marking::Reference,
```

Comment:
This change will only affect unmigrated DBs. It is a correct change to the migration, but (separately from this particular migration) is there any reasonable way to detect frontiers that were already inserted into the tree that have this issue (such as the ones that led to the bug report this PR is addressing)? Or will affected users need to recover from seed?

Reply:
The affected users will need to recover from seed, but due to the nature of the bug there should be very few affected users.